moveit_planners/chomp/chomp_interface/src/chomp_plugin.cpp

```diff
@@ -57,16 +57,10 @@ class CHOMPPlannerManager : public planning_interface::PlannerManager
     if (!ns.empty())
       nh = ros::NodeHandle(ns);
 
-    planning_interface::PlannerConfigurationMap pconfig;
     for (const std::string& group : model->getJointModelGroupNames())
     {
       planning_contexts_[group] = std::make_shared<CHOMPPlanningContext>("chomp_planning_context", group, model, nh);
-      const planning_interface::PlannerConfigurationSettings planner_config_settings{
```

Comment:
This is required for moveit_cpp to be able to find a planning pipeline as I pointed out in [moveit2#1522](https://github.com/ros-planning/moveit2/pull/1522) 
The code to find a planning pipeline is in [moveit_cpp.cpp#L167](https://github.com/ros-planning/moveit2/blob/e89526de24d88fb05eb646a7925de69ba480bfe8/moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp#L167):
```cpp
  // Retrieve group/pipeline mapping for faster lookup
  std::vector<std::string> group_names = robot_model_->getJointModelGroupNames();
  for (const auto& pipeline_entry : planning_pipelines_)
  {
    for (const auto& group_name : group_names)
    {
      const auto& pipeline = pipeline_entry.second;
      for (const auto& planner_configuration : pipeline->getPlannerManager()->getPlannerConfigurations())
      {
        if (planner_configuration.second.group == group_name) // It is required that a config for a group name exists even if it is empty
        {
          groups_pipelines_map_[group_name].insert(pipeline_entry.first);
        }
      }
    }
  }
```
I'll revert this change. This does not only affect this PR but moveit_cpp in general.

Reply:
Instead of modifying all planning plugins, you could define the empty planning config also here!
That would actually the nicest solution, because it doesn't require any changes to the planning plugins!